sdk/core/azure-core-http-netty/pom.xml

```diff
@@ -50,6 +50,7 @@
     <javaModulesSurefireArgLine>
       --add-opens com.azure.http.netty/com.azure.core.http.netty=ALL-UNNAMED
       --add-opens com.azure.http.netty/com.azure.core.http.netty.implementation=ALL-UNNAMED
+      --add-opens com.azure.core/com.azure.core.util=ALL-UNNAMED
```

Comment:
I'm intrigued on the need to add-opens for modules outside of the project, any reason this was needed?